### PR TITLE
Arg import data

### PR DESCRIPTION
**Pull request by hmc-cs-mcmurphy**
I hate github. still need help and don't know why it's making all these changes 😥 

**From before:**
I am bad at github so hopefully this doesn't ruin everything... 😟

Anyway, I've got importData.py working such that when you call python importData.py <desired path to file> it will take in that path and pass it to execute_insert. Also, I've never used args from the command line so if my style is off I'm looking for tips on that.

Again, I hope I didn't break anything in my attempt to have a branch such that it is up to date with develop.

### CODE REVIEW

**Comment by saramcallister** [comment]
Maeve, the problem appears to be that you are trying to merge the branch in master rather than develop.

File: teamworkApp/lib/importData.py [hunk 1]
```diff
@@ -1,26 +1,260 @@
...
+	conn.commit()
+	conn.close()
+
+	print("Success.")
+
+def main():
+	args = sys.argv
+	# TODO: remove this and other print statements when moving to Ruby
+	print("Inserting data)
+	execute_insert(args[1])
```
Comment (saramcallister) [inline teamworkApp/lib/importData.py]: Do we want some checks on args[1] to make sure that it is a valid input?

Comment (hmc-cs-mcmurphy) [reply teamworkApp/lib/importData.py]: Definitely will get on that thank u!

**Comment by hmc-cs-mcmurphy** [comment]
Ok, so I've gotten rid of all the wrong changes from files except for one because GitHub is insisting on adding a newline to the end of `overallBar.py` and I can't get it to keep that change. Since it doesn't affect much I'm not sure how much more I should fight to get rid of that change.

I still need to add argument validation which I will do presently. 😄

**Comment by hmc-cs-mcmurphy** [comment]
Ok, I added verification and resolved conflicts. There is still that pesky file with extra newlines... not sure how to fix it, I've tried to delete it like 5 times.

This should be good for review now! Thanks for your patience in my fight with GitHub 🤛

File: teamworkApp/lib/importData.py [hunk 2]
```diff
@@ -246,13 +247,27 @@ def execute_insert(csv_filename):
...
+		 # make sure the file is a csv
+	if filename[-4:] == ".csv":
+		# make sure the path has been properly specified
+		try:
+			execute_insert(filename)
+		except FileNotFoundError:
+			print("File must have an accessible path!")
+	else:
+		print("File must be a CSV with extension .csv!")
+
```
Comment (saramcallister) [inline teamworkApp/lib/importData.py:270]: I like this part!

**Review by saramcallister — APPROVED** [review]
Looking good :)

**Pull request merged**